### PR TITLE
feat: add page with eks iam policies and roles

### PR DESCRIPTION
**Pull request by faiq**
## Description of changes being made

Adds an additional page with missing IAM policies and roles to get EKS to work 

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4562.s3-website-us-west-2.amazonaws.com/

## Checklist

- [x] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+---
+layout: layout.pug
+navigationTitle: Explore New Cluster
+title: Explore New Cluster
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:4]: These don't look right to me...   You will also need to check the menuWeight when adding a new page.

Probably something like?

```suggestion
navigationTitle: EKS Cluster IAM Policies and Roles
title: EKS Cluster IAM Policies and Roles
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
+---
```
Comment (cneth) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:1]: The filename has a misspelling - it think you wanted 'eks-cluster-iam-policies-and-roles', not polices.  :-) 
These show up in the URI's for the page so we need to fix this...

File: pages/dkp/konvoy/2.3/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
+---
```
Comment (cneth) [inline pages/dkp/konvoy/2.3/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:1]: I don't think we want to add this page to the 2.3 github repo;  The 2.3 work on this is being done in confluence so I don't think we want it here also.

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/konvoy/2.3/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md (hunk 3 above)
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.3/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:1]: True Craig

Comment (mmcelroy91) [reply pages/dkp/konvoy/2.3/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:1]: Not a huge deal if it gets left in, but definitely need the word policies spelled right in the title from branch.  I would pull and checkout your branch, but I REALLY don't want to f-this PR up.  So, I'll wait until you have a chance Faiq.

**Review by supershal — APPROVED** [review]
Looks good to me. @faiq can you please take a look too. <--- please ignore

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    
+
+EKS IAM Artifacts
+-----------------
+
+### Policies
+
+*   `controllers-eks.cluster-api-provider-aws.sigs.k8s.io` - enumerates the Actions required by the workload cluster to create and modify EKS clusters in the user's AWS Account. It is attached to the existing `control-plane.cluster-api-provider-aws.sigs.k8s.io`  role
+    
+*   `eks-nodes.cluster-api-provider-aws.sigs.k8s.io` - enumerates the Actions required by the EKS workload cluster worker machines. It Is attached to the existing `nodes.cluster-api-provider-aws.sigs.k8s.io`
```
Comment (supershal) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:28]:
```suggestion
*   `eks-nodes.cluster-api-provider-aws.sigs.k8s.io` - enumerates the Actions required by the EKS workload cluster worker machines. It is attached to the existing `nodes.cluster-api-provider-aws.sigs.k8s.io`
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+```yaml
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  existingControlPlaneRole:
+    Type: CommaDelimitedList
+    Description: 'Names of existing Control Plane Role you want to add to the newly created EKS Managed Policy'
+    Default: control-plane.cluster-api-provider-aws.sigs.k8s.io
+  existingNodeRole:
+    Type: CommaDelimitedList
+    Description: 'ARN of the Nodes Managed Policy to add to the role'
```
Comment (supershal) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:51]:
```suggestion
    Description: 'Names of existing Node role that you want to add to the newly created EKS managed policy for nodegroups'
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+  
+Below is a [CloudFormation stack][cloudformation] that includes IAM policies and roles required to setup EKS Clusters:
+
+```yaml
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  existingControlPlaneRole:
+    Type: CommaDelimitedList
+    Description: 'Names of existing Control Plane Role you want to add to the newly created EKS Managed Policy'
```
Comment (supershal) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:47]:
```suggestion
    Description: 'Names of existing Control Plane Role you want to add to the newly created EKS Managed Policy for AWS cluster API controllers'
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+              Service:
+                - eks.amazonaws.com
+        Version: 2012-10-17
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/AmazonEKSClusterPolicy'
+      RoleName: eks-controlplane.cluster-api-provider-aws.sigs.k8s.io
+    Type: 'AWS::IAM::Role'
+```
+
+To create the resources in the cloudformation stack copy the contents above into a file and run the following command after replacing  `MYFILENAME.yaml` and `MYSTACKNAME` with the intended values:
```
Comment (supershal) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md:216]:
```suggestion
**Create the cloudformation stack using AWS CLI**:
Save above cloudformation stack into a file and run the following command after replacing  `MYFILENAME.yaml` and `MYSTACKNAME` with the intended values:
```

**Review by supershal — APPROVED** [review]
Thank you for the fixes.

**Pull request merged**